engine/src/flutter/lib/web_ui/lib/src/engine/text_editing/input_type.dart

```diff
@@ -184,5 +184,12 @@ class MultilineInputType extends EngineInputType {
   String? get inputmodeAttribute => null;
 
   @override
-  DomHTMLElement createDomElement() => createDomHTMLTextAreaElement();
+  DomHTMLElement createDomElement() => createMultilineTextArea();
+}
+
+DomHTMLTextAreaElement createMultilineTextArea() {
+  final element = createDomHTMLTextAreaElement();
+  // Scrollbar width affects text layout. This zeros out the scrollbar width.
```

Comment:
Most pedantic nit in the world, I'm sorry: 😂 

```suggestion
  // Scrollbar width affects text layout. This zeroes out the scrollbar width.
```

Reply:
Done